src/rules/__tests__/no-large-snapshots.test.ts

```diff
@@ -41,11 +41,6 @@ ruleTester.run('no-large-snapshots', rule, {
         'toThrowErrorMatchingInlineSnapshot',
       ),
     },
-    {
-      // "it should return an empty object for non snapshot files"
```

Comment:
nove it to invalid to validate it fails for non `.js`?

Reply:
I'm sorry @SimenB, I don't get your question. Could you rephrase it?

Reply:
@SimenB bumping up

Reply:
There is no test that asserts test files with any extension is tested - instead of removing the test that asserts unknown extensions are _not_ tested, change it to verify it _is_ tested